src/main/webapp/app/overview/course-card.scss

```diff
@@ -1,5 +1,8 @@
 .card {
-    min-height: 320px;
+    // best possible height to fit header, information, and manage course button: 80px header + 200px card content
+    height: 283px;
```

Comment:
80px + 200px = 280px

What are the 3 px related to?

Reply:
That additional pixel is for the course manage button, I didnt write it explicitly because it was not the case for all cards 😅

Reply:
But I can change this comment too, as I will modify some css properties of the course cards in my follow up PR